### ISSUE TITLE
🐛 [BUG] - Blog not displaying on main site

### ISSUE THREAD

**Issue by ajmarroquin** [actualbudget/docs#380]
### Description

Visiting the blog from desktop or mobile does not render the blog.

### Reproduction URL

https://actualbudget.com/blog/

### Reproduction steps

```bash
1. Go to 'actualbudget.com`
2. Click on 'Blog'
3. See Error
```

### Screenshots

```bash
<img width="768" alt="image" src="https://github.com/actualbudget/docs/assets/43194782/a96adfcd-6604-4413-bdca-f4877138e095">
```

**Comment by youngcw** [comment]
Actualbudget.com is not associated with the open source project. That's the site for the old pre open source version. I don't think that site has been updated in a few years. If you want to look at our blog look at actualbudget.org/blog

**Comment by ajmarroquin** [comment]
Thanks for the reply! I got to the repo from actualbudget.com so assumed that that was still the main site. 

I'd suggest that it would be helpful to have a blog post or somewhere on .com that points to the new home at .org! I spent an hour looking through docs on the old site and hadn't seen anywhere with the .org until your reply. I then saw it on the header of the repo, but hadn't thought to look there since I came from the .com.

Thanks for all the work, the project looks awesome and has me switching from nYNAB as a 13+ year YNABer.
